docs/data/product/dea-water-observations-statistics-landsat/_history.md

```diff
@@ -1,2 +1,6 @@
-% ## Changelog
+## Changelog
 
+### 2024-01-24: Water Observations 2023 annual summary released
+
+Product id: `ga_ls_wo_fq_cyear_3`
```

Comment:
I think you may need to add another linebreak underneath this line. Most Markdown processors interpret one linebreak as a linebreak (or as nothing) and two as a paragraph break.

Reply:
Thanks. Line break added.